### ISSUE TITLE
Buttons have no labels 🏷

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#249]
Search button doesn't have a label.
<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43589275-5a31e324-963c-11e8-8d56-3a5571e9b814.png">

Hamburger menu doesn't have a label.
<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43589605-137dadea-963d-11e8-9f50-4444437762df.png">

 'Site Privacy Protection' should be labeled properly as a toggle button. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role#Toggle_buttons
<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43589686-3cfdbb74-963d-11e8-94e1-4a5e663773b2.png">
